doc/schemas/decodepay.request.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "added": "v23.05",
+  "additionalProperties": false,
+  "required": [
+    "bolt11"
+  ],
+  "properties": {
+    "bolt11": {
+      "type": "string",
+      "added": "v23.05"
+    },
+    "description": {
+      "type": "string",
+      "added": "v23.05"
```

Comment:
These added fields are unnecessary: while these are being added to GRPC now, they existed previously.